Yoast/wordpress-seo
# Use helpcenter class for videos

## Summary

This PR can be summarized in the following changelog entry:
A new class was introduced for rendering the help center. This class is now used when possible for overall consistency with the help center.

## Test instructions

This PR can be tested by following these steps:

The following pages are edited:

- [x] google search console
- [x] metabox ( edit a post )
- [x] tax metabox ( edit a category ) 
- [x] tools: bulk editor
- [x] tools: file editor
- [x] tools: import/export

Open the help center and check if the following things are correct:

- Video
- Knowledge base search
- Go premium tab

They should all render.

Fixes #4753
